consensus/clique/src/main/java/tech/pegasys/pantheon/consensus/clique/CliqueBlockInterface.java

```diff
@@ -59,15 +59,16 @@ public Address getProposerOfBlock(final BlockHeader header) {
 
   public static BlockHeaderBuilder insertVoteToHeaderBuilder(
       final BlockHeaderBuilder builder, final Optional<ValidatorVote> vote) {
+    final BlockHeaderBuilder voteHeaderBuilder = BlockHeaderBuilder.fromBuilder(builder);
```

Comment:
I know this was a bad function - but maybe "insert" doesn't make sense given we're not changing the passed in Builder - is it simply createBuilderFrom(x, y)?

Reply:
I've gone with createHeaderBuilderWithVoteHeaders